### ISSUE TITLE
Mac (M1) shows error for every command

### ISSUE THREAD

**Issue by DanielYWoo** [telepresenceio/telepresence#2761]
**Describe the bug**
No matter what command, it says two files are missing on Mac with M1 max chips.
```
❯ telepresence connect
W0826 20:59:03.594942   34067 native_arm64.go:52] Could not read /proc/cpuinfo: open /proc/cpuinfo: no such file or directory
W0826 20:59:03.594970   34067 native_arm64.go:177] Could not read /proc/self/auxv: open /proc/self/auxv: no such file or directory
```

**Comment by ventsislav-georgiev** [comment]
It is an issue from the gvisor dependency: https://github.com/google/gvisor/issues/7849
It is safe, just an annoying warning. The patch they did is to just skip the init on non-linux systems: https://github.com/google/gvisor/pull/7850

**Comment by kreznik-augury** [comment]
Any ETA for a version with a fix?

**Comment by jsalem-brex** [comment]
[I mentioned it on the PR](https://github.com/telepresenceio/telepresence/pull/2774#issuecomment-1249662636), but the fix that was merged doesn't appear to have worked.

```
$ telepresence version
W0916 11:09:36.527997   21184 native_arm64.go:52] Could not read /proc/cpuinfo: open /proc/cpuinfo: no such file or directory
W0916 11:09:36.528028   21184 native_arm64.go:177] Could not read /proc/self/auxv: open /proc/self/auxv: no such file or directory
Client: v2.7.5 (api v3)
Root Daemon: not running
User Daemon: not running
```

Additionally, there are no release notes for 2.7.5. There is no versioning on the homebrew formula so I couldn't choose to install 2.7.4 without manually installing via `curl`.

edit: looking at the timestamps, it looks like 2.7.5 was released the same day as 2.7.4. The diff between the two commits looks like just yaml test assets.

**Comment by kreznik-augury** [comment]
It seems there was a mixup with the versions, please test v2.7.6, it should work fine
![image](https://user-images.githubusercontent.com/93582365/190888698-3c15b318-018a-49f5-b1f5-b48980f4f32b.png)

**Comment by cindymullins-dw** [comment]
Release 2.7.5 was essentially a rollback to 2.7.3 so there no release notes for it.  However we believe this issue is fixed on 2.7.6, if you could please try that version and please reopen the issue if not resolved.

**Comment by DanielYWoo** [comment]
yes, the error is gone, thanks!
